Directory.Packages.props

```diff
@@ -0,0 +1,141 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+
+    <EntityFrameworkCoreVersion>9.0.0</EntityFrameworkCoreVersion>
+    <AspNetCoreVersion>9.0.0</AspNetCoreVersion>
+    <ExtensionsVersion>9.0.0</ExtensionsVersion>
+
+    <IdentityServerVersion>4.1.2</IdentityServerVersion>
+    <MassTransitVersion>8.3.2</MassTransitVersion>
```

Comment:
PhatBoyG once wrote somewhere that this version is problematic, but I don't remember why, so we should consider bumping it on some other iteration.

Reply:
I'll bump the packages once I migrate to xUnit v3